packages/bugsnag_breadcrumbs_dart_io/README.md

```diff
@@ -0,0 +1,25 @@
+# Network Breadcrumbs for Bugsnag Flutter
+
+HTTP networking breadcrumbs for [bugsnag-flutter](https://pub.dev/packages/bugsnag_flutter) when using the `dart:io` [HttpClient](https://api.flutter.dev/flutter/dart-io/HttpClient-class.html).
+If you are using the [http](https://pub.dev/packages/http) package then [bugsnag_breadcrumbs_http](https://pub.dev/packages/bugsnag_breadcrumbs_http) should be used for networking breadcrumbs.
```

Comment:
I think you've got the dart:io and http content switched around here.

Reply:
The second line of each of these ("If you are using...") is a reference to the other package, to keep it clear that there are two ways we support this. If someone opens the `dart:io` package, I think we want them to know we also have support for `http`?

Reply:
🤦 Sorry - yes, I see: I didn't review it in context properly just noticed the reference to the other one at the end!